src/main/java/homework_1/Main.java

```diff
@@ -1,9 +1,26 @@
 package homework_1;
 
-public class Main {
-
-  public static void main(String[] args) {
-    System.out.println("Hello homework!");
-  }
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
 
-}
+public class Main {
+    public static void main(String[] args) {
+        if (Charset.defaultCharset() != StandardCharsets.UTF_8) {
+            System.out.println("For correct display cyrillic args need to run with \"-Dfile.encoding=\"UTF-8\"\"");
+        }
+        try {
+            if (args.length == 0) {
+                System.out.println("No arguments provided!");
+            } else for (String arg : args) {
+                if (arg.matches("error")) {
+                    throw new IOException("Alarm!");
+                }
+                System.out.printf("%s : %s letter(s)\n", arg, arg.length());
+            }
+        } catch (IOException e) {
+            System.err.println("Error!");
```

Comment:
Doesn't look as expected 
![image](https://user-images.githubusercontent.com/58872509/125208371-96562100-e29a-11eb-8952-8a46f23b4b90.png)

You don't need an exception, because it's not an exceptional case, it's expected behavior. And IOException is confusing here

Reply:
fixed output, removed exeption, exit loop on alarm keyword. Thought we throw exeption when we need to stop program execution.